schemas/context/person-name.schema.json

```diff
@@ -13,13 +13,13 @@
   "definitions": {
     "personname": {
       "properties": {
-        "xdm:givenName": {
-          "title": "Given name",
+        "xdm:firstName": {
+          "title": "First name",
           "type": "string",
           "description": "The personal, given, or first name."
         },
-        "xdm:surname": {
-          "title": "Surname",
+        "xdm:lastName": {
+          "title": "Last name",
```

Comment:
Ok, so this is just a title replacement? I think it is making it even more confusing. Now what is labeled `lastName` is indeed a "surname" and should be treated like one.

Reply:
@trieloff - the point of this set of changes is quite well thought out considering the complex issues. The generally agreed best principles going forward in person naming is that there is no correct way to separate person name components and that it specifically should be avoided. Hence "fullName" which is inline with these recommendations. The current overwhelming standard in the industry due to legacy RDBMS and data principles has been to have a first/last name. So that is something we cannot ignore. give/sur name actually are significant issues in their own right and just introduce a different version of first/last name problem with unique new semantics in non western cultures. 

So the root proposal is: 
a) we have a longer term correct approach with "fullName" that is more commonly being used in parallel with separated name component semantics.
b) we retain the current standard in data which is first/last name

I continue to agree with this proposal. and the interoperability it provides.

It will be interesting to see if implementation start to use "legalName", which I expect they will in some industries, and that is specifically a reason I am avoiding the "name" usage. It is very common to have a different legalName, fullName and firstName+{space}+lastName, I am an example of that. And the caucasian person who sits beside me has an issues where his firstName != givenName.

Reply:
We need to improve the descriptions.
I propose:
"The preferred first name of the person."

Reply:
+1 on @cdegroot-adobe description of the rationale.  This is not intended to be a choice of first/last versus given/surname.  We are saying that the preference is to use the full name (which is inline with the recommendations from W3C, and compatible with schema.org), but because our own applications and many applications in the industry use first/last we provide those fields to allow a direct translation.